app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,879 @@
+);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Kotlin</h3>
+        <div class="ide margin-top-small" data-lang="kotlin" data-lang-label="Kotlin SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Databases
+
+val client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+val databases = Databases(client)
+
+databases.createRelationshipAttribute(
+    databaseId = "marvel",           // Database ID                 
+    collectionId = "movies",         // Collection ID                      
+    relatedCollectionId = "reviews", // Related collection ID               
+    type = "oneToMany",              // Relationship type           
+    twoWay = true,                   // Is two-way              
+    key = "reviews",                 // Attribute key           
+    twoWayKey = "movie",             // Two-way attribute key       
+    onDelete = "cascade"             // On delete action        
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Swift</h3>
+        <div class="ide margin-top-small" data-lang="swift" data-lang-label="Swift SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+let databases = Databases(client)
+
+databases.createRelationshipAttribute(
+    databaseId: "marvel",           // Database ID              
+    collectionId: "movies",         // Collection ID            
+    relatedCollectionId: "reviews", // Related collection ID    
+    type: "oneToMany",              // Relationship type        
+    twoWay: true,                   // Is two-way               
+    key: "reviews",                 // Attribute key            
+    twoWayKey: "movie",             // Two-way attribute key        
+    onDelete: "cascade"             // On delete action         
+)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>The above example adds a relationship between the collections <b>movies</b> and <b>reviews</b>. A relationship attribute with the key <code>reviews</code> is added to the movies collection and another relationship attribute with the key <code>movie</code> is added to the reviews collection.</p>
+
+<p>Relationship attribute keys are used to reference the related collection in queries, so pick something that's intuitive and easy to remember.</p>
+
+<h2><a href="#create-documents" id="create-documents">Creating Documents</a></h2>
+<p>If a collection has relationship attributes, you can create documents in two ways.</p>
+
+<h3><a href="#create-nested" id="create-nested">Created Child Documents</a></h3>
```

Comment:
```suggestion
<h3><a href="#create-nested" id="create-nested">Creating Child Documents</a></h3>
```
In the `href` you've used `create-nested`. So do you wanna call it "Creating Nested Documents" instead ?